src/client/driver/utils/wait-for.js

```diff
@@ -0,0 +1,30 @@
+import hammerhead from '../deps/hammerhead';
```

Comment:
This could be moved to the general purpose utils. I don't like idea of local utils in general - it's easy to end up with the code duplication.